### PR TITLE
Create Metadata.json

### PR DESCRIPTION
**Pull request by ikutaa46**

### CODE REVIEW

File: items/disqus-lazy-load/metadata.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+	"name": "Disqus Lazy Load",
+	"version": "",
+	"author_username": "ikutaa46",
+	"release_date": "2018",
+	"license": "free",
+	"price_usd": "0",
+	"download_url": "https://rawcdn.githack.com/ikutaa46/ikutaa46.github.io/master/bludit/plugin/disqus-lazy-load.zip",
```
Comment (dignajar) [inline items/disqus-lazy-load/metadata.json]: The plugin is for Bludit v3.x ? if that the field for download is `download_url_v3`

File: items/disqus-lazy-load/metadata.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	"download_url_v3": "",
+	"demo_url": "",
+	"information_url": "",
+	"description": "This plugin from Bludit, edited to give site more performance. Disqus will load by default or onScrool or onClick",
```
Comment (dignajar) [inline items/disqus-lazy-load/metadata.json]: Also, I would like to improve the description, could you ?

File: items/disqus-lazy-load/metadata.json (hunk 1 above)
Comment (ikutaa46) [reply items/disqus-lazy-load/metadata.json]: i have updated this metadata.json.

1. download_url is for Bludit v2
2 download_url_v3 is for Bludit v3.

File: items/disqus-lazy-load/metadata.json (hunk 2 above)
Comment (ikutaa46) [reply items/disqus-lazy-load/metadata.json]: i only update the plugin, i dont know what to say at description, u may improve it.

**Pull request closed without merging**